test-case/check-audio-equalizer.sh

```diff
@@ -44,103 +45,100 @@ func_test_eq()
     local id=$1
     local conf=$2
 
-    [ ! -f $conf ] && dloge "$conf not exist" && exit 1
+    [ ! -f "$conf" ] && dloge "$conf not exist" && exit 1
```

Comment:
I don't think this logic is exist, base on `FilterList=($(ls -d "${my_dir}"/eqctl/eq_"${testfilter}"_*.txt))`